MOBILE_API.xml

```diff
@@ -4902,6 +4902,13 @@
                 If omitted on supported displays, the displayed alert shall not have any SoftButtons.
             </description>
         </param>
+
+        <param name="alertIcon" type="Image" mandatory="false" since="6.0">
```

Comment:
Needs to be added to `ImageFieldName` as well

Reply:
added in c126332

Reply:
Need to run RPCParser again to update README

Reply:
ran again in [459b3c7](https://github.com/smartdevicelink/rpc_spec/pull/180/commits/459b3c7b5b2aad4ff56d5d896b2931ca42165726)